winpr/include/winpr/nt.h

```diff
@@ -1267,7 +1267,7 @@ static INLINE NTSTATUS NTSTATUS_FROM_WIN32(long x)
 }
 #endif
 
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(__MINGW32__)
```

Comment:
Unnecessary

Reply:
This is needed - For MS tools you undefine and redeclare FILE_INFORMATION_CLASS and it's enum constants.
In MingW they already defined so all those tricks shall be disabled, with only winintr include remaining. Overwise you get double declaration errors for constants(You undefine only the enum itself, not constants, remember?)